Geometry/test/testGeometryQC8GE21.py

```diff
@@ -3,15 +3,15 @@
 process = cms.Process('TestBeam')
 
 # import of standard configurations
-#process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.Services_cff')
 process.load('FWCore.MessageService.MessageLogger_cfi')
-#process.load('Configuration.EventContent.EventContent_cff')
-#process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
 process.load('gemsw.Geometry.GeometryQC8GE21_cff')
-#process.load('Configuration.StandardSequences.MagneticField_0T_cff')
-#process.load('Configuration.StandardSequences.Reconstruction_cff')
-#process.load('Configuration.StandardSequences.RecoSim_cff')
-#process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.MagneticField_0T_cff')
+process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('Configuration.StandardSequences.RecoSim_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
```

Comment:
why do you load these??

Reply:
@manuelrodriguezgiraldo Could you comment out these lines like `testGeometryTestBeam.py`?
https://github.com/gem-sw/gemsw/blob/main/Geometry/test/testGeometryTestBeam.py

Reply:
at one point I was not loading the geometry and I was checking if it worked and I forgot to remove this

Reply:
remove please